### PR TITLE
doc: write out the fee recommendation

### PR DESCRIPTION
**Pull request by patricklodder**
A document containing the fee recommendation in its current form, including derived default parametrization for mining inclusion and mempool parametrization. This gives us a nice summary of the current policy. 

I had a bigger document at first, but figured that it's a lot easier to read if there's not much to read.

### CODE REVIEW

File: doc/fee-recommendation.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+wallet, are:
+
+- **0.01 DOGE per kilobyte** transaction fee
+- **1.00 DOGE** dust limit (discard threshold)
+- **0.001 DOGE** replace-by-fee increments
+
+The wallet rejects transactions that have outputs under the dust limit, and
+discards change to fee if it falls under this limit.
+
+Note: the recommended dust limit is expected to be lowered in a follow-up
```
Comment (rnicoll) [inline doc/fee-recommendation.md:24]:
```suggestion
Note: The recommended dust limit is expected to be lowered in a follow-up
```
Nit but consistent capitalisation with below

**Review by rnicoll — DISMISSED** [review]
One very minor nit, then good to merge

File: doc/fee-recommendation.md (hunk 1 above)
Comment (patricklodder) [reply doc/fee-recommendation.md]: ✅

thanks!

**Review by rnicoll — APPROVED** [review]
LGTM

**Comment by CHANDRAMA2** [comment]
Please sir Return my dogecoin

**Pull request merged**